prifi-lib/client/client.go

```diff
@@ -588,3 +640,16 @@ func (p *PriFiLibClientInstance) Received_REL_CLI_TELL_EPH_PKS_AND_TRUSTEES_SIG(
 
 	return nil
 }
+
+// ValidateHash returns true iff both hashes passed to the fnction are equal
+func ValidateHash(hash_relay, hash_client []byte) bool {
```

Comment:
There is the builtin function `Bytes.Equal()` which does this. https://golang.org/pkg/bytes/#Equal